mini_pupper_control/mini_pupper_control/StanfordQuadruped/pupper/HardwareInterface.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2023 stanfordroboticsclub
```

Comment:
It is defined in mini_pupper_bsp or mini_pupper_2_bsp, so I think this file is not necessary.